src/config/database.yml

```
@@ -1,8 +1,13 @@
 default: &default
   adapter: postgresql
+  encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: <%= ENV.fetch('POSTGRES_USER', 'postgres') %>
+  password: <%= ENV.fetch('POSTGRES_PASSWORD', 'root') %>
```

[Q] this isn't true for the default setup of postgres
```suggestion
  password: <%= ENV['POSTGRES_PASSWORD'] %>
```